### ISSUE TITLE
Error while running `make all`(CentOS 7)

### ISSUE THREAD

**Issue by sandoche2k** [zonemaster/zonemaster-cli#170]
```sh
make all
cp lib/Zonemaster/CLI.pm blib/lib/Zonemaster/CLI.pm
: share/Makefile
cd share && make all
make[1]: Entering directory `/home/sandoche/zonemaster-cli/share'
Makefile:6: *** missing separator.  Stop.
make[1]: Leaving directory `/home/sandoche/zonemaster-cli/share'
make: *** [pm_to_blib] Error 2
```

**Comment by matsduf** [comment]
There is already a solution for this. See PR #169.

@sandoche2k, please test and review.

**Comment by matsduf** [comment]
Resolved by #169.
